CalculatorTests/Tests.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using NUnit.Framework;
+using static Calculator.Program;
+
+namespace CalculatorTests
+{
+    [TestFixture]
+    public class Tests
+    {
+        [Test]
+        public void AddOneNumber()
```

Comment:
Currently only tests involving addition exist, if these were extended to include all operators and some of the additional methods, such as IsValidOperator, checking for digits, division by Zero, it would make us more confident in the refactoring.